plotly/tools.py

```diff
@@ -1507,26 +1563,28 @@ def create_quiver(x, y, u, v, scale=.1, arrow_scale=.3,
             for more information on valid kwargs call
             help(plotly.graph_objs.Scatter)
 
-        :rtype (trace): returns quiver trace
+        :rtype (dict): returns a representation of quiver figure.
 
         Example 1: Trivial Quiver
         ```
-        # 1 Arrow from (0,0) to (1,1)
+        import plotly.plotly as py
+        from plotly.tools import FigureFactory as FF
 
         import math
 
-        quiver = TraceFactory.create_quiver(x=[0], y=[0],
-                                            u=[1], v=[1],
-                                            scale=1)
+        # 1 Arrow from (0,0) to (1,1)
+        fig = FF.create_quiver(x=[0], y=[0],
```

Comment:
:cow2: I think being explicit will be helpful to users? `FF` only makes sense to us because we've been talking about `FigureFactory` for a while now?